posts/2025-07-01-25.0.0.7-beta.adoc

```diff
@@ -0,0 +1,301 @@
+// Replace navaneethsnair1 with your GitHub username eg: lauracowen
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+// Replace Navaneeth S Nair with your name as you'd like it to be displayed,
+// eg: LauraCowen
+//
+// For every link starting with "https://openliberty.io" in the post make sure to use
+// {url-prefix}. e.g- link:{url-prefix}/guides/GUIDENAME[GUIDENAME]:
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE CAPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+BETA_RELEASE_SUMMARY
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to an introductory paragraph. This sentence is really
+// important because it is supposed to grab the readers attention.  Make sure to keep the blank lines
+//
+// Throughout the doc, replace 25.0.0.7-beta with the version number of Open Liberty, eg: 22.0.0.2-beta
+// // // // // // // //
+
+The link:{url-about}[Open Liberty] 25.0.0.7-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<SUB_TAG_0, MicroProfile OpenAPI 4.1>>
+* <<SUB_TAG_1, MicroProfile 7.1>>
+* <<SUB_TAG_2, MicroProfile Telemetry 2.1>>
+
+// // // // // // // //
+// In the preceding section:
+// Change SUB_FEATURE_TITLE to the feature that is included in this release and
+// change the SUB_TAG_1/2/3 to the heading tags
+//
+// However if there's only 1 new feature, delete the previous section and change it to the following sentence:
+// "The link:{url-about}[Open Liberty] 25.0.0.7-beta includes SUB_FEATURE_TITLE"
+// // // // // // // //
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/31811
+// Contact/Reviewer: Azquelt
+// // // // // // // // 
+[#SUB_TAG_0]
+== MicroProfile OpenAPI 4.1
+
+
+OpenAPI is a standardised way of documenting REST APIs in a JSON or YAML format. MicroProfile OpenAPI helps you generate and serve OpenAPI documentation for your REST applications built using JAX-RS or Jakarta restfulWS. This can be useful for developers to test out the API during development, or for people using the API in production.
```

Comment:
Could you move MP OpenAPI 4.1 to the bottom please? It's the least interesting and impactful change in the list.